### DISCUSSION TITLE
Change font

### DISCUSSION THREAD

**Discussion by delta0815** [owncast/owncast#4059] [Q&A] [answered]
Hi all,
I was wondering if it's possible to change the font of my owncast site so I added the following code into the CSS box:

@font-face {
    font-family: "Waltograph";
    url("https://www.rundisney.de/img/emoji/Waltograph.woff2")format("woff2"),
}
:root {
  --font-family: 'Waltograph', sans-serif;
}

The font is available on the site and when I load the page I even see this customization in the HTML code but it didn't change anything on the website itself.

Any idea?

Thanks and best regards,

quicky

**Comment by gabek** [comment, marked as answer]
There is no variable named `--font-family`. Try using `--theme-text-display-font-family` or `--theme-text-body-font-family` depending on your needs.

https://owncast.online/components/?path=/docs/owncast-styles-typography--docs
